deploy/dependencies.yml

```diff
@@ -32,9 +35,27 @@
         - postfix
         - unzip
         - p7zip-full
-        - npm
+        - build-essential
+        - libssl-dev
 
-    - file: src=/usr/bin/nodejs dest=/usr/local/bin/node state=link
+    - name: ensure nvm install file is removed
+      file: path="/tmp/install.sh" state=absent force=true
+
```

Comment:
It would be nice to see these tasks conditional on something. e.g. already installed, correct version, or if the download exists, that kind of thing.  Maybe a simple check as the first task that registers a var to see if nvm is installed and then have the other tasks conditional on that.